regolith/helpers/Google_Calendar_API/credentials.json

```diff
@@ -0,0 +1 @@
+{"installed":{"client_id":"154057909362-e30si590e8n6n6p4lljth074kj7dgv9o.apps.googleusercontent.com","project_id":"regolith-addprojectum-calendar","auth_uri":"https://accounts.google.com/o/oauth2/auth","token_uri":"https://oauth2.googleapis.com/token","auth_provider_x509_cert_url":"https://www.googleapis.com/oauth2/v1/certs","client_secret":"gmxkVOmAn5tY8SVJTMswtIpG","redirect_uris":["urn:ietf:wg:oauth:2.0:oob","http://localhost"]}}
```

Comment:
what is this?  I am guessing it shouldn't be here.

It will live forever in the git history, so if this is a valid token it should be changed on the Google side so it doesn't give access anywhere.

Reply:
This is a credentials file that contains client_id and client_secret that allows people to access the api. https://cloud.google.com/docs/authentication/end-user The documentation here says this json file has to be distributed with the application. This other documentation https://developers.google.com/identity/protocols/oauth2 says: 'The process results in a client ID and, in some cases, a client secret, which you embed in the source code of your application. (In this context, the client secret is obviously not treated as a secret.)' However, I'm not an expert with this, so distributing this info does seem kind of insecure to me.